src/java.base/share/classes/java/lang/reflect/Proxy.java

```diff
@@ -878,7 +878,7 @@ private static void ensureVisible(ClassLoader ld, Class<?> c) {
             }
             if (type != c) {
                 throw new IllegalArgumentException(c.getName() +
-                        " referenced from a method is not visible from class loader");
+                        " referenced from a method is not visible from class loader: " + ld.toString());
```

Comment:
Isn't the `.toString()` call redundant in string concatenation?

Reply:
Thanks @liach for reviewing, Fixed this.